src/Airbrake/Notifier.php

```diff
@@ -144,7 +144,7 @@ public function sendNotice($notice)
     {
         foreach ($this->filters as $filter) {
             $notice = $filter($notice);
-            if ($notice === false) {
+            if ($notice === null || $notice === false) {
```

Comment:
I'm curious, can't we just check it like this `if (!$notice)`? Yes, that would also pass if `$notice === 0`, but it seems like it doesn't really matter, since we only care if the value is falsey.